### PR TITLE
Passive

### PR DESCRIPTION
**Pull request by rodsol**

### CODE REVIEW

**Comment by AmeBel** [comment]
what does `(.*)` mean in https://github.com/opencog/relex/blob/master/data/relex2logic-rules.txt#L209

@williampma

**Comment by williampma** [comment]
regex string match for matching anything before "passive", so "past-passive", "present-passive", etc. all works.

**Comment by linas** [comment]
"." means "match anything" and "*" means "match it zero or more times".

File: opencog/nlp/relex2logic/rules/passive.scm [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                (VariableNode "$word")
+                (DefinedLinguisticConceptNode "person")
+                
+            )
+            (InheritanceLink 
+                (VariableNode "$verb")
+                (VariableNode "$tense")
+            )
+		    (EvaluationLink
+                (DefinedLinguisticRelationshipNode "_obj")
```
Comment (AmeBel) [inline opencog/nlp/relex2logic/rules/passive.scm]: whitespace correction here.

**Comment by AmeBel** [comment]
has merge conflict pls resolve

File: opencog/nlp/relex2logic/loader/load-rules.scm [hunk 2]
```diff
@@ -61,3 +61,4 @@
 (load-scm-from-file "../opencog/nlp/relex2logic/rules/gender.scm")
 (load-scm-from-file "../opencog/nlp/relex2logic/rules/conj.scm")
 (load-scm-from-file "../opencog/nlp/relex2logic/rules/tense.scm")
+(load-scm-from-file "../opencog/nlp/relex2logic/rules/passive.scm")
```
Comment (linas) [inline opencog/nlp/relex2logic/loader/load-rules.scm]: this is where the merge conflict is. The path has been shortened.

**Pull request merged**